static/js/util/profile_edit.js

```diff
@@ -62,7 +69,7 @@ export function boundRadioGroupField(keySet: string[], label: string, options: O
     );
   });
 
-  const value = _.get(profile, keySet);
+  const value = String(_.get(profile, keySet));
```

Comment:
I'm curious about this overall, is there a reason that you can't use a boolean as the value of the options we provide to the radio button select? why do we need to convert it to a string?